tests/geometryRTheta/advection_rtheta/README.md

```diff
@@ -10,8 +10,8 @@ The tests are made for different parameters which are:
 ### - The mapping and the domain used for the advection: 
  - Circular mapping in the physical domain (CircularToCartesian and AdvectionPhysicalDomain); 
  - Czarny mapping in the physical domain (CzarnyToCartesian and AdvectionPhysicalDomain); 
- - Czarny mapping in the pseudo-Cartesian domain (CzarnyToCartesian and AdvectionPseudoCartesianDomain); 
- - Discrete mapping of the Czarny mapping in the pseudo-Cartesian domain (DiscreteToCartesian and AdvectionPseudoCartesianDomain).
+ - Czarny mapping in the pseudo-Cartesian domain (CzarnyToCartesian); 
+ - Discrete mapping of the Czarny mapping in the pseudo-Cartesian domain (DiscreteToCartesian).
```

Comment:
Why do you remove `AdvectionPseudoCartesianDomain` on these lines but not `AdvectionPhysicalDomain` on the lines above?